extension/src/NABfunctions.ts

```diff
@@ -1469,3 +1470,38 @@ export async function troubleshootParseAllFiles(): Promise<void> {
   }
   logger.show();
 }
+export async function convertFromTemplate(
+  extensionUri: vscode.Uri
+): Promise<void> {
+  logger.log("Running: convertFromTemplate");
+  Telemetry.trackEvent("convertFromTemplate");
+  try {
+    const workspaceFolderPath = SettingsLoader.getWorkspaceFolderPath();
+    const templateSettingsFilePath = path.join(
+      workspaceFolderPath,
+      "template.json"
+    );
+    if (!fs.existsSync(templateSettingsFilePath)) {
+      throw new Error(
+        `The template settings file "${templateSettingsFilePath}" was not found.`
```

Comment:
Ran into this error while running the test app. I think we could provide a better message and hint.
![image](https://user-images.githubusercontent.com/17023248/151929696-efa7517f-bfce-40c1-add9-12422ff767d1.png)

Reply:
This shouldn't ever run on an existing app so we might need to block/inform the user?

Reply:
Better now?

Reply:
missing commit?

Reply:
aah, sorry. I had another change that I didn't want to commit in that file, so I missed this one. (I've commented out the part where the VSCode editor is reloaded, since it breaks debugging...)

Reply:
Clear as day :+1:

Reply:
With a lot of repeating "Project", I just noticed. 🙂
But that might be ok...